src/OctoshiftCLI.IntegrationTests/BbsToGithub.cs

```diff
@@ -117,7 +120,7 @@ await retryPolicy.Retry(async () =>
             _tokens.Add("AWS_ACCESS_KEY_ID", Environment.GetEnvironmentVariable("AWS_ACCESS_KEY_ID"));
             _tokens.Add("AWS_SECRET_ACCESS_KEY", Environment.GetEnvironmentVariable("AWS_SECRET_ACCESS_KEY"));
             var awsBucketName = Environment.GetEnvironmentVariable("AWS_BUCKET_NAME");
-            archiveUploadOptions = $" --aws-bucket-name {awsBucketName} --aws-region {AWS_REGION}";
+            archiveUploadOptions = $" --aws-bucket-name {awsBucketName} --aws-region {AWS_REGION} --use-github-storage {useGithubStorage}";
```

Comment:
There are a couple of things here:
1. `--use-github-storage` is not yet a supported option for the `generate-script` commands so this ultimately won't work until we add the option.
2. Even if the GitHub storage option was supported by `generate-script`  this still wouldn't work because both AWS and GitHub storage options are provided at the same time which should be a validation error.

Reply:
Ok so are you saying we cant have integration tests?

Reply:
Removing int test logic for now